### PR TITLE
Note recursive objects in Security Considerations

### PR DESCRIPTION
**Pull request by strugee**
As discussed on IRC.

I'm not sure this is the best wording, but it seemed decent, so...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2489,6 +2489,16 @@ <h3 property="schema:name">Social Web Working Group</h3>
         </p>
       </section>
 
+      <section id="security-recursive-objects">
+        <h2>Recursive Objects</h2>
+        <p>
+	  Servers should take care to properly handle recursive ActivityStreams
```
Comment (jaywink) [inline index.html]: I think it might be a good idea to open up "properly". What should the implementer do to protect themselves? Or a short example of what could happen if not. At least I'm not sure what I should do to fulfil this paragraph.

Comment (strugee) [reply index.html]: @jaywink perhaps something like:

> Servers should set a limit on how deep to recurse while resolving objects, or otherwise specially handle ActivityStreams objects with recursive references.

Does that seem better to you? It's a bit wordy but I feel like we shouldn't imply that setting a recursion limit is the only way to handle this; there are probably more intelligent things you can do

Comment (jaywink) [reply index.html]: I think it's better as it's a bit more concrete on the problem and how to handle it 👍

**Comment by strugee** [comment]
Pushed a new commit with new wording as discussed with @jaywink :thumbsup:
